### ISSUE TITLE
Double tap to zoom doesn't work in PDF viewer

### ISSUE THREAD

**Issue by mneuwert** [owncloud/ios-app#163]
In the PDF viewer, you can use pinch-to-zoom gestures but double-tap to change between zoomed out and zoomed in states doesn't work. It seems to be a PDFKit issue in conjunction with `PDFView` configured to be used with `UIPageViewController`. If this configuration is not used, double-tap-to-zoom works.

### Steps to reproduce
1. Open PDF document
2. Double-tap on the currently seen page

### Expected behaviour
PDFView should zoom in and center the spot user double-tapped.

### Actual behaviour
Nothing happens

### Client
**iOS version: 12.x**

**ownCloud app version: latest master**

**Device model: iPhone Xs Max, iPhone 8 Simulator**

**Comment by michaelstingl** [comment]
@mneuwert Please check the details, then we can decide the next steps…

**Comment by mneuwert** [comment]
@michaelstingl I conducted some testing with following outcome:
- I think by default PDFView doesn't implement double tap to zoom at least from I have seen in my debugging session.
- I tried to implement double tap to zoom and managed to make it work (sort of) but the problem is that go(rect, page) called on PDFView after zooming in to actually force the view to show the part of the page user double-tapped on doesn't seem to work. So, it would always zoom to the middle of the page.
- This is actually how double-tap to zoom works in our Image Viewer.. For which I will create another issue since we can definetely do that better there since there is zoom(to:rect) method available on UIScrollView..
